packages/slice-machine/src/components/HoverCard/HoverCard.stories.tsx

```diff
@@ -63,33 +63,3 @@ export const Video = {
     ),
   },
 } satisfies Story;
-
-export const CloseWhenClickingOutSide = {
```

Comment:
Is this no longer a requirement?

Reply:
Yep finally :/

https://prismic-team.slack.com/archives/C02L3FN3AJK/p1690529552348829